randomize

```diff
@@ -1035,6 +1041,11 @@ if ('inBin' in argv) {
         process.exit(1)
       }
     }
+    if (!argv.accessibilityPatches) {
+      // Apply accessibility patches
+      const result = applyAccessibilityPatches()
+      check.apply(result)
```

Comment:
Doesn't this influence the checksum if randomizing from command line?

Reply:
I don't see how it would. The checksum has already been awaited and verified before this. I tested both browser and cli, but will do more just in case.

Reply:
No, that's my mistake, I failed to notice that you're applying this after assigning the `checksum` variable.